tests/Feature/CycleMenuNotifyCommandTest.php

```
@@ -11,121 +13,129 @@
-        'time_of_day' => '12:30:00',
-        'quantity' => '1 wrap',
-        'position' => 1,
-    ]);
-
-    return $menu->refresh();
-}
+    public function test_cycle_menus_notify_today_dry_run_outputs_payload_and_does_not_send_notifications(): void
+    {
+        // Given an active menu and at least one user
+        User::factory()->create();
+        $menu = $this->seedActiveMenuForToday();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$menu'. (undefined)

(UnusedLocalVariable)